flow360/component/simulation/models/material.py

```diff
@@ -0,0 +1,67 @@
+from typing import Literal
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.base_model import Flow360BaseModel
+
+
+class MaterialBase(Flow360BaseModel):
+    # Basic properties required to define a material.
+    # For example: young's modulus, viscosity as an expression of temperature, etc.
+    ...
+    name: str = pd.Field()
+
+
+class Gas(MaterialBase):
+    # contains models of getting properites, for example US standard atmosphere model
+    name: Literal["gas"] = pd.Field(frozen=True)
+    viscosity_type: Literal["Sutherland", "constant"] = pd.field("Sutherland")
+    constant_dynamic_viscosity: Optional[ViscosityType] = pd.Field(None)
+    sutherland_constants: Optional[SutherlandConstants] = pd.Field(None)
+    specific_heat_ratio: float = pd.Field()
+    gas_constant: GasConstantType = pd.Field()
+    Prandtl_number: float = pd.Field()
```

Comment:
let's not capitalise after name

Reply:
You mean I shouldn't capitalize Prandtl? It is conventionally capitalized though. We also have `Mach`

Reply:
yes, this is what I mean. Let's not capitalise after name. It should be `mach`, `reynolds`, `prandtl`